pages/thc/[processRef]/review.tsx

```diff
@@ -207,24 +152,16 @@ const ReviewPage: NextPage = () => {
         The Tenancy and Household Check has now been completed. Please review
         the answers with all present tenants and ask them to sign.
       </Paragraph>
-      {tenantsPresent.length > 0 && (
-        <Paragraph>Present for check: {tenantsPresent.join(", ")}</Paragraph>
-      )}
-      {sections
-        .filter((section) => section.rows.length)
-        .map(({ heading, rows }) => (
-          <React.Fragment key={heading}>
-            <Heading level={HeadingLevels.H2}>{heading}</Heading>
-
-            {rows.length ? (
-              <SummaryList
-                rows={(rows as unknown) as { key: string; value: string }[]}
-              />
-            ) : (
-              <Paragraph>Loading...</Paragraph>
-            )}
-          </React.Fragment>
-        ))}
+      <Paragraph>
+        Present for check:{" "}
+        {presentTenantNames.length > 0
+          ? presentTenantNames.join(", ")
+          : "Loading..."}
+      </Paragraph>
```

Comment:
I reckon this can be generated by the `IdAndResidencyReviewSection` now, so let's do that.

Reply:
As discussed, we're leaving this as the review section just has the IDs, which would duplicate the lookups for names (as we need to pass that data through to the ReviewSection anyway)